spot-ml/src/main/scala/org/apache/spark/sql/WideUDFs.scala

```diff
@@ -16,8 +16,8 @@
  */
```

Comment:
Maybe we could open a user story about this to be done after the release? I did a little bit of research but I think is going to take some time to discuss this and make it work. 

What I found is that the org.apache.spark.sql.functions.udf function will allow you to create a udf with max 10 parameters and SparkSession.udf.register function allow you to create a udf with max 22 parameters.